lib/classes/CLI.js

```diff
@@ -149,6 +149,13 @@ class CLI {
     const command = this.serverless.pluginManager.getCommand(commandsArray);
     const commandName = commandsArray.join(' ');
 
+    // Remove entrypoint subcommands
+    _.forEach(command.commands, (details, cmd) => {
```

Comment:
PS: I've written this out of my mind - and did not test it ;-)
Please check if there is an issue. In general the approach should work.